aiosmtpd/smtp.py

```diff
@@ -132,8 +132,11 @@ def _client_connected_cb(self, reader, writer):
         self._writer = writer
 
     def eof_received(self):
+        super().eof_received()
```

Comment:
This looks weird.  Why do you need this overridden method at all if you're just going to upcall to super?